lib/sonar/builders/docker.py

```diff
@@ -47,7 +47,10 @@ def docker_build(
         )
 
         client = docker_client()
-        return client.images.get(image_name)
+        image = client.images.get(image_name)
+        logger.info("Successfully build docker-image, SHA256: {}".format(image.id))
```

Comment:
lets add the sha as an attribute in the trace - unfortunately logs are not saved globally for search.

https://ui.honeycomb.io/mongodb-4b/environments/production/datasets/evergreen-agent/trace/e8Wt6WdtQ26?fields[]=s_name&fields[]=s_serviceName&span=166e96577d7e3ad8


example: https://github.com/mongodb/mongodb-kubernetes/blob/3178929a901f7bf30090fae7f634043c07b0db65/pipeline.py#L375-L380

then you can see it under your linked trace in the build
```

Reply:
nice - seems to work 
https://ui.honeycomb.io/mongodb-4b/environments/production/result/2NA2RSzW66K